docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -14,23 +18,29 @@ These methods are useful if you still want to keep the module separate from the
 
 ## Use a monorepo
 
-For this guide, we assume that you have the following project structure:
+Your project should use the following structure:
+
+- **apps/**: Store multiple projects here, including React Native apps.
+- **packages/**: Keep different packages used by your apps in this folder.
+- **package.json**: This is the root package file that contains the Yarn workspaces configuration.
```

Comment:
```suggestion
- **apps**: A directory to store multiple projects, including React Native apps.
- **packages**: A directory to keep different packages used by your apps.
- **package.json**: This is the root package file that contains the Yarn workspaces configuration.
```

Instead of using a slash at the end of directory names, let's remove it and spell it out. It is always helpful for users to read rather than see a symbol and wonder what it is meant for.